tests/integration/components/permission-language-test.js

```diff
@@ -0,0 +1,25 @@
+import { moduleForComponent, test } from 'ember-qunit';
+import hbs from 'htmlbars-inline-precompile';
+
+moduleForComponent('permission-language', 'Integration | Component | permission language', {
+  integration: true
+});
+
+test('it renders', function(assert) {
```

Comment:
Hmmm. I think those tests are probably not what we need. Go ahead and take them out, and then I'll approve the PR and get you working on something else.